src/nvim/tui/tui.c

```diff
@@ -779,6 +779,9 @@ static void fix_terminfo(TUIData *data)
   } else if (STARTS_WITH(term, "screen")) {
     unibi_set_if_empty(ut, unibi_to_status_line, "\x1b_");
     unibi_set_if_empty(ut, unibi_from_status_line, "\x1b\\");
+  } else if (STARTS_WITH(term, "tmux")) {
```

Comment:
Sure.  But I am not sure if it's really correct, and therefore just copy'n'pasted it.

Some documentation here (on the function and the vars) would be good.